src/Utilities/RalewayText.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import { Text } from 'react-native';
+
+export class RalewayText extends Component {
```

Comment:
I can't wait to hear about the lengthy process of getting this working!  Nice job.